src/lib.rs

```diff
@@ -1476,11 +1476,6 @@ impl EGraph {
         self.type_info.get_sort_by(pred)
     }
 
-    /// Returns a sort based on the type
```

Comment:
@Alex-Fischman Actually, is there a reason for removing this convenient method (`get_sort`)? Found myself using it today but it went missing.

Reply:
I thought it was unused, you can add it back